cross/cython/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME = cython
+PKG_VERS = 0.29.21
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
```

Comment:
This has been inserted to ensure that locally the cache file is persisted with an explicit name.